Dockerfile-prod

```diff
@@ -24,12 +24,10 @@ COPY .env* ./
 # Building the image
 RUN npm run build
 
-# Copy env files
-COPY .env* ./
-
 # Then, use nginx to serve the built files
 # See https://hub.docker.com/_/nginx
 FROM nginx:alpine
 
 # Copying the built files into the nginx image, to the default served directory
 COPY --from=build /usr/src/app/build /usr/share/nginx/html
+COPY ./nginx-config/nijobs.conf /etc/nginx/conf.d/default.conf
```

Comment:
Why have a directory? Also, the `./` in the copy from looks weird, and is unnecessary.
The reason I see would be to override all files in the directory, but that only applies using volumes so it would be the same to just `RUN rm /etc/nginx/conf.d/default.conf` and then `COPY nijobs-nginx.conf /etc/nginx/conf.d/whatever.conf`

Reply:
It is not necessary to have a rm + cp. Instead we just cp to the default, which will override and work, since it's in an isolated environment, it won't cause a problem when overriding.

Reply:
Sure. However, using a directory just for holding an nginx conf file seems weird to me